eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -70,8 +70,9 @@ const AttendanceCard: React.FC<Props> = ({
         <UserAttendance
           key={user.customer_id}
           slotId={id}
-          attended={!absentees?.includes(user.customer_id)}
+          attended={attendance?[user.customer_id].attended}
```

Comment:
This is probably a TS error as `boolean` is expected and you're passing a `string | null` do the `Boolean()` assertion